docs/examples/plotting_functions/scatter.md

```diff
@@ -105,6 +104,106 @@ f
 ```
 \end{examplefigure}
 
+### Bezier path markers
+
+You can also use bezier paths as markers.
+A bezier path marker should fit into the square from -1 to 1 in x and y to be comparable in size to other default markers and to be correctly rendered by GLMakie, because here the path has to be rendered to a bitmap first .
```

Comment:
I didn't have any problems rendering a 5..25 x 5..25 square in GLMakie so I think this is wrong. Or are there issues with more complex shapes?

Reply:
Oh that's probably outdated, I started out implementing it that way, but later changed it so each marker is scaled to fit before rendering to the atlas.